cronlock

```diff
@@ -130,9 +127,8 @@ function raw_cmd {
       fi
       ;;
     *) # net yet handled
-      echo "ERR - Unknown response" >&2
+      fatal "ERR - Unknown response"
       info "${response}"
```

Comment:
this line needs a look

Reply:
Yeah I actually saw this and just ignored it. I was thinking of either just keeping the fatal and removing the info line. Or just remove the `info` line entirely. They both exit the script.

Don't really need two log lines.

Reply:
Ok great :+1: